config/manifests/bases/self-node-remediation.clusterserviceversion.yaml

```diff
@@ -6,10 +6,10 @@ metadata:
     capabilities: Basic Install
     categories: OpenShift Optional
     containerImage: quay.io/medik8s/self-node-remediation-operator:latest
-    createdAt: "2022-09-11 12:36:32"
+    createdAt: "2022-11-06 08:57:52"
     description: Self Node Remediation Operator for remediate itself in case of a
       failure.
-    olm.skipRange: '>=0.1.0 <0.4.0'
+    olm.skipRange: '>=0.1.0 <0.5.0'
```

Comment:
Does SNR have v0.4.0 or lower? I think only PP, thus we can potentially use `v0.4.0` as the minimum version for upgrades.